### PR TITLE
feat(core): flatten structure of core package

### LINKED ISSUE #139: Refactor `core` package
## Main issues

Currently, the `core` package is a mess, and is in great need of a refactor. I've outlined some main issues:

- [x] The file structure does not make sense. Things are not where you would think
- [x] We output several empty `.css`-files on build
- [x] Several rules are written multiple times in the final `core.css` file

## Suggestions

Here are some suggested steps to alleviate these issues:

- [x] Move to a flat structure (as already discussed) for _some of_ the files. Specifically, move all files that will have content when parsed on build, to root level
- [x] Let all "pure `scss`" files reside in folders and not get built. This would be most files in the `variables` folder, along with mixins and functions
- [x] Clean up the `@import` chain in all files to remove duplicate code exports
- [ ] Export custom properties for the variables

### PR DESCRIPTION
**Pull request by piofinn**
affects: @fremtind/jkl-accordion, @fremtind/jkl-bullet-list, @fremtind/jkl-button,
@fremtind/jkl-card, @fremtind/jkl-checkbox, @fremtind/jkl-core, @fremtind/jkl-divider-line,
@fremtind/jkl-dropdown, @fremtind/jkl-footer, @fremtind/jkl-grid, @fremtind/jkl-header,
@fremtind/jkl-logo, @fremtind/jkl-radio-button, @fremtind/jkl-text-input

## 📥 Proposed changes

Flatten structure of `core` package as proposed in issue #139 

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING]() doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

Fixes #139

### CODE REVIEW

File: packages/bullet-list/example/index.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "~@fremtind/jkl-core/src/typography/root.scss";
+@import "~@fremtind/jkl-core/core.scss";
```
Comment (lfbergee) [inline packages/bullet-list/example/index.scss]: import the entire core?

Comment (lfbergee) [reply packages/bullet-list/example/index.scss]: Ah, its example, thats fine

File: packages/core/package.json [hunk 2]
```diff
@@ -18,10 +18,9 @@
     ],
     "scripts": {
         "build": "yarn build:styles && yarn build:modules",
-        "build:css": "node-sass src -o build/css --importer='../../node_modules/node-sass-tilde-importer'",
+        "build:styles": "node-sass . -o . --non-recursive --importer='../../node_modules/node-sass-tilde-importer'",
         "prebuild:scss": "mkdirp -p build && mkdirp build/scss",
```
Comment (lfbergee) [inline packages/core/package.json]: This can go

File: packages/core/package.json [hunk 3]
```diff
@@ -18,10 +18,9 @@
...
         "build:scss": "cpx 'src/**/*.scss' build/scss",
```
Comment (lfbergee) [inline packages/core/package.json]: This too.

File: packages/radio-button/radio-button.scss [hunk 4]
```diff
@@ -1,5 +1,4 @@
-@import "~@fremtind/jkl-core/build/scss/typography/font-size.scss";
-@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+@import "~@fremtind/jkl-core/variables/variables";
```
Comment (lfbergee) [inline packages/radio-button/radio-button.scss]: Not a huge fan of the semantic `/variables/variables`.

Doesn't _index.scss work here? So you can do `@import "~@fremtind/jkl-core/variables";`

**Comment by lfbergee** [comment]
Also cpx-depdendency can be deleted

File: packages/core/package.json (hunk 2 above)
Comment (piofinn) [reply packages/core/package.json]: Yep! That was me not remembering the underscore thing in Sass 😅

Comment (piofinn) [reply packages/core/package.json]: Yep! And `--non-recursive` can be removed as well

File: packages/radio-button/radio-button.scss (hunk 4 above)
Comment (piofinn) [reply packages/radio-button/radio-button.scss]: Maybe? I agree that it would be better, so I'll check it out!

Comment (piofinn) [reply packages/radio-button/radio-button.scss]: Seems like this was only implemented in LibSass in May, and is not a part of node-sass yet. Should be right around the corner, though!

https://github.com/sass/node-sass/issues/2339

File: packages/bullet-list/example/index.scss (hunk 1 above)
Comment (piofinn) [reply packages/bullet-list/example/index.scss]: Yeah, I mostly just wanted to make sure it worked both ways!

**Comment by piofinn** [comment]
> Also checkout the _index.scss file instead of _variables.scss and _mixin.scss

Not possible yet in node-sass as of yet (new feature in LibSass from May)

File: packages/radio-button/radio-button.scss (hunk 4 above)
Comment (lfbergee) [reply packages/radio-button/radio-button.scss]: okay, how about all.scss?

import variables/all.scss?

**Pull request merged**